.husky/commit-msg

```diff
@@ -0,0 +1 @@
+npx --no -- commitlint --edit $1
```

Comment:
Sorry, what is this for?

Reply:
it comes from docs https://commitlint.js.org/guides/local-setup.html:

>yarn add --dev husky

>yarn husky init

> # Add commit message linting to commit-msg hook
> echo "yarn commitlint --edit \$1" > .husky/commit-msg

Reply:
the `--no --` part ensures that commitlint runs without any global npm configurations interfering, and --edit $1 passes the commit message file to commitlint for validation.